docs/features/latency.md

```diff
@@ -0,0 +1,224 @@
+General statement about latency
+===================================
```

Comment:
```suggestion
# General statement about latency
```

Reply:
Right, forgot about this. This document must have been created a long time ago.

Reply:
```suggestion
## General statement about latency
```